### ISSUE TITLE
S3 Permission Denied when attempting to Exztract qt5-install

### ISSUE THREAD

**Issue by ghost** [daleglass/vircadia-builder#77]
NOte: I'm building on an 18.04/Ubuntu system.

During the "Downloading Qt from AWS" portion of the build, I received a "Connection reset by peer"
Checking the S3/URL, I noted:

This XML file does not appear to have any style information associated with it. The document tree is shown below.
<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>806Q918P9KQ03XW0</RequestId>
<HostId>NBnCleigpSEo8fw7yOWRZe2LkaL7BQHBsn08JJrUrq3DcPGy5Fs5FuxXu0B5b7Mj0trBtHG8TNk=</HostId>
</Error>

**Comment by daleglass** [comment]
By the way, is there a reason you need to specifically build on 18.04? Starting with 21.04 you can build against the system Qt packages.

I recall there being some issues with npm, but I thought that was solved at this point. Yeah, please open a separate issue on that.

**Comment by daleglass** [comment]
Fair enough! Please also report issues with the AppImage, it should work correctly.

**Comment by JulianGro** [comment]
To be fair, 18.04 is what we build the releases on, so it is the most well tested and supported.

Also yeah, there shouldn't be any missing dependencies. Are you sure your 18.04 install is up to date?

**Comment by mwbogucki** [comment]
Julian,
No worries... I appreciate you guys making an effort to help out.
Here's one thing I did...
I mounted the app-image using: $> ./Vircadia-x86_64_v2021.1.3-Eos.AppImage
--appimage-mount /tmp/mount_myXXXX

/tmp/.mount_VircadKWV9jE
***@***.***:[/tmp/.mount_VircadKWV9jE]
----------------------------------------------------------------------
[16:46:05]
$> ll
total 0
drwxr-xr-x  5 root root   0 Sep  8 12:14 usr
drwxr-xr-x 13 root root   0 Sep  8 12:14 qt5-install
lrwxrwxrwx  1 root root  47 Sep 18 21:16 Vircadia.png ->
usr/share/icons/hicolor/16x16/apps/Vircadia.png
-rw-rw-r--  1 root root 142 Sep 18 21:16 Vircadia.desktop
lrwxrwxrwx  1 root root  12 Sep 18 21:16 .DirIcon -> Vircadia.png
lrwxrwxrwx  1 root root  28 Sep 18 21:16 AppRun ->
vircadia/interface/interface
drwxr-xr-x  9 root root   0 Sep 18 21:16 vircadia

Under ./usr/ ... I see the following subdirs:  bin lib64 share

No "lib"

The error I'm seeing is
./Vircadia-x86_64_v2021.1.3-Eos.AppImage:
/usr/lib/x86_64-linux-gnu/libQt5Core.so.5

I'm wondering if I need to symbolically link lib64 to lib...

Will monkey around with it.. to see if that may be the case.

--Mike

**Comment by mwbogucki** [comment]
Success!!! I invoked the following:

$>export LD_LIBRARY_PATH=/tmp/ ./Vircadia-x86_64_v2021.1.3-Eos.AppImage

Sorry for all the confusion gentlemen.

--Mike

**Comment by JulianGro** [comment]
> I recall there being some issues with npm, but I thought that was solved at this point. Yeah, please open a separate issue on that.

@mwbogucki https://github.com/vircadia/vircadia/issues/1228#issuecomment-848808726
